docs/apache-airflow/production-deployment.rst

```diff
@@ -749,6 +749,102 @@ additional apt dev and runtime dependencies.
     --build-arg ADDITIONAL_RUNTIME_APT_DEPS="default-jre-headless"
 
 
+Actions executed at image start
+-------------------------------
+
+If you are using the default entrypoint of the production image,
+there are a few actions that are automatically performed when the container starts.
+In some cases, you can pass environment variables to the image to trigger some of that behaviour.
+
+Creating system user
+....................
+
+Airflow image is Open-Shift compatible, which means that you can start it with random user ID and group id 0.
+Airflow will automatically create such a user and make it's home directory point to /home/airflow.
+You can read more about it in `Openshift best practices <https://docs.openshift.com/container-platform/4.1/openshift_images/create-images.html#images-create-guide-openshift_create-images>`_
+in the "Support arbitrary user ids" chapter.
+
+Verify Airflow DB connection
+............................
+
+In case Postgres or MySQL DB is used, the entrypoint will wait until the airflow DB connection becomes
+available. This happens always when you use the default entrypoint.
+
+Upgrading Airflow DB
+....................
+
+If you set ``_AIRFLOW_DB_UPGRADE`` variable to a non-empty value, the entrypoint will perform
```

Comment:
@potiuk Can you tell me why you decided to prefix the airflow variables with an `_`? Are you just not sure if this is the right way to do this? (i.e are they not worth to be that "public" :D)

If you omit the underscore they're still different to airflow (config) env variables. I just would like to know :)

Reply:
The main reason is that it is a "behavior" change and it is not "AIRFLOW build time behavior but container "execution" behaviour. All the other AIRFLOW_* variables impact the way how the image is built, but those have no impact on the image, but change how container behaves.  So I wanted to have them clearly separated so that they are not easily confused. 

Naming is the most difficult part (as usual) - so happy to change it (better idea?) or add some explanation why they are named like that.

WDYT @feluelle ?

Reply:
Added explanation now, but I am open to any suggestions :)

Reply:
SGTM 👍 Makes sense. Thanks for the explanation.